src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/deciders/Decider.java

```diff
@@ -90,4 +95,31 @@ public void handleNodeMuted() {
 
   @Override
   public abstract Decision operate();
+
+  /**
+   * read threshold values from rca.conf
+   *
+   * @param conf RcaConf object
+   */
+  @Override
+  public void readRcaConf(RcaConf conf) {
+    configObj = conf.getDeciderConfig();
+  }
+
+  public double getFieldDataCacheUpperBound() {
+    return configObj.fieldDataCacheUpperBound;
+  }
+
+  public double getShardRequestCacheUpperBound() {
+    return configObj.shardRequestCacheUpperBound;
+  }
+
+  public List<String> getWorkLoadPriority() {
+    return configObj.workloadPriorityOrder;
+  }
+
+  public List<String> getCachePriority() {
+    return configObj.cachePriorityOrder;
+  }
```

Comment:
Left a comment on DeciderConfig as well. Please use private members + getters for this

Reply:
Done